src/http_fork_source.rs

```diff
@@ -0,0 +1,53 @@
+use zksync_web3_decl::{
+    jsonrpsee::http_client::{HttpClient, HttpClientBuilder},
+    namespaces::{EthNamespaceClient, ZksNamespaceClient},
+};
+
+use crate::fork::{block_on, ForkSource};
+
+#[derive(Debug)]
+/// Fork source that gets the data via HTTP requests.
```

Comment:
nit: Add a description/comment for `fork_url`. Ex: RPC Url for network to fork

This is becoming increasingly necessary as we're now hosting the Rust docs : https://matter-labs.github.io/era-test-node/era_test_node/index.html